src/main/java/network/brightspots/rcv/GuiConfigController.java

```diff
@@ -1253,8 +1281,27 @@ private void loadConfig(ContestConfig config) {
         .setText(rules.multiSeatBottomsUpPercentageThreshold);
     textFieldDecimalPlacesForVoteArithmetic.setText(rules.decimalPlacesForVoteArithmetic);
     textFieldMinimumVoteThreshold.setText(rules.minimumVoteThreshold);
-    textFieldMaxSkippedRanksAllowed.setText(rules.maxSkippedRanksAllowed);
-    textFieldMaxRankingsAllowed.setText(rules.maxRankingsAllowed);
+    if (rules.maxSkippedRanksAllowed
+        .equalsIgnoreCase(ContestConfig.MAX_SKIPPED_RANKS_ALLOWED_UNLIMITED_OPTION)) {
+      checkBoxMaxSkippedRanksAllowedUnlimited.setSelected(true);
```

Comment:
Do you need `checkBoxMaxSkippedRanksAllowedUnlimited.setDisable(false)` here also? Not sure why this is different from what you're doing for `checkBoxMaxRankingsAllowedMax` on line 1297.

Reply:
No, I don't think it's needed. The difference is this field will never have the check box disabled, whereas the other one you mentioned is disabled when no `WinnerElectionMode` is selected.